packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -301,29 +327,33 @@ describe('plugin-conversation', () => {
 
         describe('jumpToActivity()', () => {
           it('should throw an error if search object is missing', () => {
-            const {jumpToActivity} = webex.internal.conversation.listActivitiesThreadOrdered({conversationUrl: convoUrl});
+            const {jumpToActivity} = webex.internal.conversation.listActivitiesThreadOrdered({
+              conversationUrl: convoUrl,
+            });
 
-            try {
-              jumpToActivity();
-            }
-            catch (e) {
-              assert.equal(e.message, 'Search must be an activity object from conversation service');
-            }
+            jumpToActivity().catch((e) => {
+              assert.equal(
+                e.message,
+                'Search must be an activity object from conversation service'
+              );
+            });
           });
 
           it('should throw an error if activity.target.url is missing', () => {
-            const {jumpToActivity} = webex.internal.conversation.listActivitiesThreadOrdered({conversationUrl: convoUrl});
+            const {jumpToActivity} = webex.internal.conversation.listActivitiesThreadOrdered({
+              conversationUrl: convoUrl,
+            });
 
-            try {
-              assert.throws(jumpToActivity({target: null}));
-            }
-            catch (e) {
+            jumpToActivity({target: null}).catch((e) => {
```

Comment:
same again, before this PR this test would fail if `jumpToActivity()` didn't fail, now it won't fail